tests/ecmp/test_fgnhg.py

```diff
@@ -0,0 +1,333 @@
+import pytest
+from datetime import datetime
+
+import time
+import logging
+import ipaddress
+import json
+from tests.ptf_runner import ptf_runner
+from tests.common import config_reload
+
+# Constants
+NUM_NHs = 8
+DEFAULT_VLAN_ID = 1000
+DEFAULT_VLAN_IPv4 = ipaddress.ip_network(u'200.200.200.0/28')
+DEFAULT_VLAN_IPv6 = ipaddress.ip_network(u'200:200:200:200::/124')
+PREFIX_IPv4 = u'100.50.25.12/32'
+PREFIX_IPv6 = u'fc:05::/128'
+ARP_CFG = '/tmp/arp_cfg.json'
+FG_ECMP_CFG = '/tmp/fg_ecmp.json'
+USE_INNER_HASHING = False
+NUM_FLOWS = 1000
+
+SUPPORTED_TOPO = ['t0']
+SUPPORTED_PLATFORMS = ['mellanox']
+
+logger = logging.getLogger(__name__)
+
+def configure_interfaces(cfg_facts, duthost, ptfhost, ptfadapter, vlan_ip):
+    config_port_indices = cfg_facts['port_index_map']
+    port_list = []
+    eth_port_list = []
+    ip_to_port = {}
+    bank_0_port = []
+    bank_1_port = []
+
+    # remove existing IPs from PTF host
+    ptfhost.script('scripts/remove_ip.sh')
+    # set unique MACs to PTF interfaces
+    ptfhost.script('scripts/change_mac.sh')
```

Comment:
@anish-n can we use this pattern `from tests.common.fixtures.ptfhost_utils import change_mac_addresses      # lgtm[py/unused-import]` ?